tls/s2n_resume.c

```diff
@@ -29,6 +29,9 @@
 #include "tls/s2n_crypto.h"
 #include "tls/s2n_tls.h"
 
+#define MIN_TLS13_SECRET_LEN 28
+#define MAX_TLS13_SECRET_LEN 48
+
```

Comment:
Isn't MAX_TLS13_SECRET_LEN (and maybe MIN_TLS13_SECRET_LEN?) already defined elsewhere in the code?